packages/core/src/browser/preferences/preference-service.ts

```diff
@@ -27,6 +27,7 @@ export interface PreferenceService extends Disposable {
     get<T>(preferenceName: string): T | undefined;
     get<T>(preferenceName: string, defaultValue: T): T;
     get<T>(preferenceName: string, defaultValue?: T): T | undefined;
+    set(preferenceName: string, value: string, scope: Scope): void;
```

Comment:
it would be nice it `scope` can be omitted with the default to `Workspace` preference scope.

Reply:
Don't need this method any more